libs/pjproject/patches/150-config_site.patch

```diff
@@ -11,34 +11,23 @@
 + * Since both pjproject and asterisk source files will include config_site.h,
 + * we need to make sure that only pjproject source files include asterisk_malloc_debug.h.
 + */
-+
-+/* #if defined(MALLOC_DEBUG) && !defined(_ASTERISK_ASTMM_H)
-+ * #include "asterisk_malloc_debug.h"
-+ * #endif
-+ */
++#if 0 && !defined(_ASTERISK_ASTMM_H)
```

Comment:
This shouldn't get in. We compile pjproject standalone, not from within Asterisk, so it makes not sense to have something in this file that references asterisk sources (even if it'll not get used anyway).

Reply:
The patch was created from the [config_site.h file from asterisk](https://github.com/asterisk/asterisk/blob/master/third-party/pjproject/patches/config_site.h). All changed to this patch are because of asterisk commits, see [here](https://github.com/asterisk/asterisk/commits/master/third-party/pjproject/patches/config_site.h).
The only change I did was that 'if 0' to not use the asterisk header (it was already there, just commented), because I tried to make the diff against the upsteam file as minimal as possible. That'll make future updates easier.